Dockerfile

```diff
@@ -1,14 +1,14 @@
-FROM ubuntu:focal-20221019
+FROM ubuntu:focal-20221130
 
-ARG VERSION=15.6.2
+ARG VERSION=15.7.0
 
 ENV GITLAB_VERSION=${VERSION} \
     RUBY_VERSION=3.0.4 \
     RUBY_SOURCE_SHA256SUM="70b47c207af04bce9acea262308fb42893d3e244f39a4abc586920a1c723722b" \
     GOLANG_VERSION=1.18.8 \
     GITLAB_SHELL_VERSION=14.13.0 \
```

Comment:
This should be updated to 14.14.0 (see <https://gitlab.com/gitlab-org/gitlab-foss/-/blob/v15.7.0/GITLAB_SHELL_VERSION>).